token/program-2022-test/tests/confidential_transfer.rs

```diff
@@ -890,22 +913,6 @@ async fn confidential_transfer_deposit() {
             )
         )))
     );
-
-    token
-        .confidential_transfer_apply_pending_balance(&alice_meta.token_account, &alice, 0, 65537, 2)
-        .await
-        .unwrap();
-
-    let state = token
-        .get_account_info(&alice_meta.token_account)
-        .await
-        .unwrap();
-    let extension = state
-        .get_extension::<ConfidentialTransferAccount>()
-        .unwrap();
-    assert_eq!(extension.pending_balance_credit_counter, 0.into());
-    assert_eq!(extension.expected_pending_balance_credit_counter, 2.into());
-    assert_eq!(extension.actual_pending_balance_credit_counter, 2.into());
```

Comment:
These asserts seem important -- did you mean to add them back in?

Reply:
Oh yes, I added these back in. There was actually a small defining the maximum deposit amount constant, so I fixed that as well.